comp-sci/data-structures-and-algorithms/intro-graphs/the-components-of-a-graph-ii.md

```diff
@@ -0,0 +1,80 @@
+---
+author: mihaiberq
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: must-know
+
+aspects:
+
+  - introduction
+
+  - workout
+
+parent: the-graph-data-structure
+
+---
+
+# The Components of a Graph
+
+---
+## Content
+
+A graph is **connected** if between every vertex in the graph, there is a *path*. The graph in the previous insight[1] was connected, while the one below isn't (you cannot get from *A* to *B*, for example):

```

Comment:
can you guys think of any better way to reference a footnote? @Stefan-Stojanovic  @nem035 
```suggestion
A graph is **connected** if between every vertex in the graph, there is a *path*. The graph in this footnote[1] is connected, while the one below isn't (you cannot get from *A* to *B*, for example):
```